website/docs/troubleshooting.md

```diff
@@ -23,3 +23,15 @@
 
 - If you recieve errors complaining about DRI3 settings, please reference issue
   [#44](https://github.com/neovide/neovide/issues/44#issuecomment-578618052).
+
+- If your scroling is very stutter
+
+    - Add flag "--novsync" before startup as a quickfix.
```

Comment:
Lately I've seen some users use ligatures in the browser, maybe having `--novsync` in a codeblock as well isn't too bad.